spec/models/web_of_science_source_record_spec.rb

```diff
@@ -26,6 +26,26 @@
     end
   end
 
+  context 'sets identifiers' do
+    before do
+      identifiers = WebOfScience::Identifiers.new(wos_record)
+      allow(identifiers).to receive(:doi).and_return('doi')
+      allow(identifiers).to receive(:pmid).and_return('123')
+      allow(WebOfScience::Identifiers).to receive(:new).and_return(identifiers)
+    end
+    it 'sets doi' do
+      expect(wos_src_rec.doi).to eq 'doi'
+    end
+    it 'sets pmid' do
+      expect(wos_src_rec.pmid).to eq 123
+    end
+    it 'allows select' do
+      expect(wos_src_rec.save!).to be true
```

Comment:
Not going to block over it, but since we aren't intending to test the return of `.save!` here, the following lines are equivalent:
```ruby
expect(wos_src_rec.save!).to be true
wos_src_rec.save!
```
In either case the exclamation point is the, err... point.  The test is invalid without the successful save.

Reply:
Thanks, sorry I missed that.  I want to fix all the places where that happens.